tests/integration/release/test_release.py

```diff
@@ -82,26 +85,33 @@ def test_release_row_noising(
     engine = get_engine_from_string(engine_name)
 
     data_file_paths = get_dataset_filepaths(Path(source), dataset_schema.name)
-    filters = get_generate_data_filters(dataset_schema, year, state)
-    unnoised_data = [load_standard_dataset(path, filters, engine) for path in data_file_paths]
+    filters = get_data_filters(dataset_schema, year, state)
+    unnoised_data: list[pd.DataFrame | dd.DataFrame] = [
+        load_standard_dataset(path, filters, engine) for path in data_file_paths
+    ]
 
     if engine == DASK_ENGINE:
         # TODO: [MIC-5960] move this compute to later in the code
         dataset_data: list[pd.DataFrame] = [data.compute() for data in unnoised_data if len(data) != 0]  # type: ignore [operator]
     else:
         dataset_data = [data for data in unnoised_data if len(data) != 0]  # type: ignore [misc]
 
-    has_small_shards = dataset_name == "acs" or dataset_name == "cps" or dataset_name == "wic"
-    if str(source) == RI_FILEPATH and has_small_shards:
-        dataset_data = [pd.concat(dataset_data).reset_index()]
-
-    datasets = [Dataset(dataset_schema, data, SEED) for data in dataset_data]
+    seed = SEED
+    if dataset_schema.name != DatasetNames.CENSUS and year is not None:
+        seed = seed * 10_000 + year
```

Comment:
I wouldn't change it, but can't you also just do `seed += year`?

Reply:
That would work for this particular case, but I wanted to make it obvious that this code was taken from the generate_data functions.

Reply:
`seed = hash(f"{seed}_{year}")` might be a clearer way to achieve the same.